pom.xml

```diff
@@ -1277,7 +1277,7 @@
               <exclude>**/.settings/**/*</exclude>
 
               <!-- Proto/grpc generated wrappers -->
-              <exclude>**/sdks/python/apache_beam/runners/api/*.py</exclude>
+              <exclude>**/apache_beam/runners/api/*_pb2*.py</exclude>
```

Comment:
I agree with both comments.

I think we can create core/runners and move api and portability there. Since these are marked experimental, it is fine to move them. Other stuff needs to be shelved until 3.0